### DISCUSSION TITLE
Pre-install and post- install handler doén't work correct

### DISCUSSION THREAD

**Discussion by gianghm1408** [rauc/rauc#1543] [Q&A]
I add  Pre-install and post- install handler to my system.config as bellow :

``` ini
[handlers]
pre-install = /etc/rauc/scripts/pre_install.sh
post-install = /etc/rauc/scripts/post_install.sh
```

pre_install.sh : 
``` sh
 #!/bin/bash

ECHO_TTY="/dev/ttyS2"
bootpart_value=$(fw_printenv bootpart)

if [ "$bootpart_value" = "bootpart=0:4" ]; then
  #  unmount /dev/mmcblk0p3 và /dev/mmcblk0p
  echo start  umount RootA  >> ${ECHO_TTY}
  umount /dev/mmcblk0p3 >> ${ECHO_TTY}
  sleep 20
  echo finish umount Root >> ${ECHO_TTY}
  umount /dev/mmcblk0p5 >> ${ECHO_TTY}
  sleep 5
  echo finish umount App >> ${ECHO_TTY}
else
  # unmount /dev/mmcblk0p4 và /dev/mmcblk0p6
  echo start  umount RootB  >> ${ECHO_TTY}
  umount /dev/mmcblk0p4 >> ${ECHO_TTY}
  sleep 20
  echo finish umount Root >> ${ECHO_TTY}
  umount /dev/mmcblk0p6 >> ${ECHO_TTY}
  sleep 5
  echo finish umount App >> ${ECHO_TTY}
fi
```

When I install bundle I jusst seen "echo" and "sleep" cmd work and don't umount partiton But when I run pre_install.sh  by my seft It
work correctly and can umount partiton. Please help me fix It

**Comment by ejoerns** [comment]
You do not see any output or you don't see the actual mounts happen?

Also note that stderr will not be redirected to ${ECHO_TTY} this way. Thus you won't see any errors.
Maybe try pattern

    umount /dev/mmcblk0p3 >> ${ECHO_TTY} 2>&1

to also redirect stderr to stdout.

However, this is not really related to rauc probably.

**Comment by gianghm1408** [comment]
I don't see the actual umount happen. And after try with new pattern "umount /dev/mmcblk0p3 >> ${ECHO_TTY} 2>&1" I still didn't see any error. I know this issue may note relate to RAUC but I think some one in community  may meet same issue with me and can help to resolve it so I push to Q&A
